src/libraries/System.Formats.Tar/src/System/Formats/Tar/TarEntry.cs

```diff
@@ -98,7 +101,10 @@ public DateTimeOffset ModificationTime
             get => _header._mTime;
```

Comment:
Can't comment above this line: Can you please also update the exception message in line 95?